keyboards/cmm_studio/saka68/hotswap/config.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
> add sam's sg81m keyboard

but has other changes outside that keyboard?

Reply:
No. The saka has already done in the previous or. This pr only contains changes to the Sg81m keyboard

Reply:
![image](https://user-images.githubusercontent.com/16520359/105916223-84ecc580-6028-11eb-90aa-7ae852debf2e.png)

clearly contains other files

Reply:
@zvecr probably I accidentally pressed enter to that line and split /* Copyright line into two lines. Reverted the change by making it back to one line and hopefully, it is ok now. thanks for pointing it out